common/src/main/java/org/jboss/hal/testsuite/page/runtime/DomainDeploymentPage.java

```diff
@@ -102,4 +105,16 @@ public Column navigateToDeploymentColumnInServerGroup(String groupName) {
         Console.withBrowser(browser).waitUntilLoaded();
         return column;
     }
+
+    public List<String> getDeploymentBrowsedContentItems() {
```

Comment:
This code duplication between `DeploymentPage` and `DomainDeploymentPage` will be fixed in another PR. You don't need to fix this.